src/hpc/thread.c

```diff
@@ -191,7 +213,19 @@ static NOINLINE void GrowStack(void)
 
 static NOINLINE void SetupTLS(void)
 {
-#ifndef HAVE_NATIVE_TLS
+#if defined USE_MACOS_PTHREAD_TLS_ASM
```

Comment:
So `USE_MACOS_PTHREAD_TLS_ASM` is not "native" TLS, but also not `USE_PTHREAD_TLS`? Yet it shares the name with the latter, and a lot of the code with the former (e.g. in `AllocateTLS`) ? This is rather confusing, isn't it? Or maybe I already got it wrong (which, however, would kinda underline my point ;-).

Reply:
Correct, but also gone now.